docs/src/main/sphinx/admin/properties-query-management.rst

```diff
@@ -12,16 +12,27 @@ Query management properties
 Configures the algorithm to organize the processing of all of the
 stages of a query. You can use the following execution policies:
 
-* ``phased`` schedules stages in a sequence to avoid blockages because of 
-  inter-stage dependencies. This policy maximizes cluster resource utilization 
+* ``phased`` schedules stages in a sequence to avoid blockages because of
+  inter-stage dependencies. This policy maximizes cluster resource utilization
   and provides the lowest query wall time.
-* ``all-at-once`` schedules all of the stages of a query at one time. As a 
-  result, cluster resource utilization is initially high, but inter-stage 
-  dependencies typically prevent full processing and cause longer queue times 
+* ``all-at-once`` schedules all of the stages of a query at one time. As a
+  result, cluster resource utilization is initially high, but inter-stage
+  dependencies typically prevent full processing and cause longer queue times
   which increases the query wall time overall.
-* ``legacy-phased`` has similar functionality to ``phased``, but can increase 
+* ``legacy-phased`` has similar functionality to ``phased``, but can increase
   the query wall time as it attempts to minimize the number of running stages.
 
+
+``query.hash-partition-count``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Default value:** ``100``
+* **Session property:** ``hash_partition_count``
+
+The number of partitions to use for processing distributed joins and
+aggregations.
```

Comment:
Do we want to offer advice on what higher and lower numbers mean .. e.g. make the number to low and query processing will not be able to parallelize enough. Make the number too high and ?? .. also how much should it correspond with number of workers (I think not at all but...)

Reply:
I wouldn't offer advice yet since we don't have any data on that yet. Ideally it should be adaptive or driven by CBO

Reply:
ok